Update high_level_multiplayer.rst (#7932)
godotengine/godot-docs: tutorials/networking/high_level_multiplayer.rst

must have the same name. When using ``add_child()`` for nodes which are expected to use RPCs, set the argument
``force_readable_name`` to ``true``.

.. warning:: 

>     If a function is annotated rpc on the client script (resp. server script), then this function must also be declared on the server script (resp. client script), and both
>     must have the same signature, even if this function is not currently used.
> 
>     If these conditions are not fulfilled, the script may raise an error or cause unwanted behavior. See further explanation and troubleshoot on [this post](https://github.com/godotengine/godot/issues/57869#issuecomment-1034215138).

Reviewer: ```suggestion
    If a function is annotated with ``@rpc`` on the client script (resp. server script), then this function must also be declared on the server script (resp. client script), and both
    must have the same signature, **even if this function is not currently used**.

    If these conditions are not fulfilled, the script may print an error or cause unwanted behavior. See further explanation and troubleshooting on [this post](https://github.com/godotengine/godot/issues/57869#issuecomment-1034215138).
```

>     If these conditions are not fulfilled, the script may raise an error or cause unwanted behaviour. See further explanation and troubleshoot on [this post](https://github.com/godotengine/godot/issues/57869#issuecomment-1034215138).

Reviewer 2: ```suggestion
    If these conditions are not fulfilled, the script may raise an error or cause unwanted behavior. See further explanation and troubleshoot on [this post](https://github.com/godotengine/godot/issues/57869#issuecomment-1034215138).
```
Use US english

        var sender_id = multiplayer.get_remote_sender_id()
        # Process the input and affect game logic.

>     
> 

Reviewer: ```suggestion
```
